tony-cli/src/test/java/com/linkedin/tony/TestClusterSubmitter.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2018 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.tony;
+
+import com.linkedin.tony.cli.ClusterSubmitter;
+import org.testng.annotations.Test;
+
+import static junit.framework.Assert.*;
```

Comment:
Let's not mix both junit and testng in the same test. It looks like the rest of TonY uses testng, so let's stick with that.